src/keyboard.cpp

```diff
@@ -0,0 +1,196 @@
+#include "keyboard.h"
+#include "aldente.h"
+#include "physics.h"
+#include "scene/scene_camera.h"
+
+bool Keyboard::keys[1024];
+bool Keyboard::lmb_down;
+bool Keyboard::rmb_down;
+bool Keyboard::mouse_moved;
+glm::vec3 Keyboard::last_cursor_pos;
+
+// TODO : This class won't be used in the final game anyways.
```

Comment:
However, it will be useful for debugging when we don't have controllers readily available. Good to keep this around.